--- ./imap/src/osdep/unix/ssl_unix.c.org	2010-10-02 10:37:57.000000000 +0200
+++ ./imap/src/osdep/unix/ssl_unix.c	2011-05-12 16:42:14.772573334 +0200
@@ -28,14 +28,14 @@
 
 #define crypt ssl_private_crypt
 #define STRING ssl_private_STRING
-#include <x509v3.h>
-#include <ssl.h>
-#include <err.h>
-#include <pem.h>
-#include <buffer.h>
-#include <bio.h>
-#include <crypto.h>
-#include <rand.h>
+#include <openssl/x509v3.h>
+#include <openssl/ssl.h>
+#include <openssl/err.h>
+#include <openssl/pem.h>
+#include <openssl/buffer.h>
+#include <openssl/bio.h>
+#include <openssl/crypto.h>
+#include <openssl/rand.h>
 #undef crypt
 #undef STRING
 
